### PR TITLE
Add markUnSeen in translations

### PR DESCRIPTION
**Pull request by asgalecki**
https://user-images.githubusercontent.com/70109479/178031355-ac9d6194-8ee5-414b-897f-2c1fd8b8cfd6.mp4

https://user-images.githubusercontent.com/70109479/178031364-c54e5e23-9218-406c-b80e-2c6e8fda2080.mp4

### CODE REVIEW

**Review by lukbar48 — APPROVED** [review]

**Review by lukbar48 — APPROVED** [review]

**Comment by asgalecki** [comment]
https://user-images.githubusercontent.com/70109479/178031270-19c07344-8565-4a9a-9358-add2389dad8a.mp4


https://user-images.githubusercontent.com/70109479/178031287-b90bf1e2-f9f1-4b2c-9207-bf8941ca6b7c.mp4

File: src/screens/dashboard/components/Notification.tsx [hunk 1]
```diff
@@ -11,13 +11,24 @@ import { notificationNavHandler } from '../helpers/notificationNavHandler'
...
+  const [opacity, setOpacity] = useState(defaultOpacity)
   const endDate = 'endDate' in p ? new Date(p.endDate) : undefined
   const description = 'description' in p ? p.description : undefined
   const { user } = useUserContext()
   const { navigate } = useNavigation()
   const { mutate } = useMarkNotificationAsSeen()
-  const opacity = wasSeenByHolder ? 0.6 : 1
+
+  useEffect(() => {
+    if (wasSeenByHolder) setOpacity(0.6)
```
Comment (lukbar48) [inline src/screens/dashboard/components/Notification.tsx]: setOpacity(wasSeenByHolder ? 0.6 : 1)

File: src/screens/dashboard/components/Notification.tsx [hunk 2]
```diff
@@ -54,34 +65,36 @@ const Notification = ({ source: author, wasSeenByHolder, type, ...p }: Notificat
...
+              <Text variant="textXS" color="darkGreyBrighter">
+                {formatDate(new Date(p.createdAt), 'ago')}
+              </Text>
+            </Box>
           </Box>
-        </Box>
-      </BaseOpacity>
+        </BaseOpacity>
+      </Box>
       {modalUser && (
```
Comment (lukbar48) [inline src/screens/dashboard/components/Notification.tsx:95]: maybe its possible to move this modal component inside Box and remove fragments?

File: src/data-access/mutations/useMarkNotificationAsUnseen.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import { useMutation } from 'react-query'
+import axios, { AxiosError } from 'axios'
+import { Notification } from 'mockApi/models'
+import { API } from '../API'
+import { queryClient } from '../queryClient'
+import { QueryKeys } from '../QueryKeys'
+
+export type MarkUnseenResponse = { notification: Notification }
+
+const markNotificationAsUnseen = async (id: string): Promise<MarkUnseenResponse> => {
```
Comment (lukbar48) [inline src/data-access/mutations/useMarkNotificationAsUnseen.ts]: I think this file can be combined with useMarkNotificationsAsSeen file as it has similar responsibility, just to stay consistent with other mutation hook files

**Review by lukbar48 — APPROVED** [review]

File: src/data-access/mutations/useMarkNotificationAsUnseen.ts (hunk 3 above)
Comment (asgalecki) [reply src/data-access/mutations/useMarkNotificationAsUnseen.ts]: Good point, indeed.

**Pull request merged**